lib/twofishes/result.rb

```diff
@@ -16,17 +14,12 @@ def self.from_response(hash)
       end
     end
 
-    def name
-      feature.name
-    end
-
-    def display_name
-      feature.display_name
-    end
-
-    def country_code
-      feature.cc
-    end
+    def_delegator :@data, :what
```

Comment:
I make it that way first, but i looked like:
```
def_delegators :@data, :what, :where, :feature
def_delegators :feature, :name, :display_name
def_delegator :feature, :cc, :country_code
```
What could be confusing, that's why I decide to write each delegation in separated line.